### PR TITLE
Add: minor improvements for assertions

### PR DESCRIPTION
**Pull request by AnthonyMadia**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Lint (`yarn lint --check`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- Add minor improvements to a few unit tests.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/frontend/src/__tests__/blockButton.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+        <UserContext.Provider value={userData}>
+            <BlockButton type={type} count={count} data={data} />
+        </UserContext.Provider>
+    )
+}
+
+test('should render BlockButton props correctly with comment button', () => {
     // Mocked props
     const type = 'comment'
     const count = 1
```
Comment (vimwitch) [inline packages/frontend/src/__tests__/blockButton.js:17]: Can we change this count variable to something more unique? I guess it’s unlikely that the number 1 is hanging around in the document, but it’s less likely that 274728 is in the document.

File: packages/frontend/src/__tests__/signupPage.js [hunk 2]
```diff
@@ -28,18 +39,19 @@ test('should render SignupPage correctly with content[0]', () => {
...
-    )
+    renderSignupPage(userData, postData)
     expect(
         screen.getByText(
             /currently, UniRep Social is an invite only community. Please enter your invitation code below./i
         )
     ).toBeInTheDocument()
+    expect(screen.getByText(/join us/i)).toBeInTheDocument()
+    // generate text on signup page
+    const textbox = screen.getByRole('textbox')
```
Comment (vimwitch) [inline packages/frontend/src/__tests__/signupPage.js:54]: Maybe use a variable here

**Review by vimwitch — APPROVED** [review]

File: packages/frontend/src/__tests__/signupPage.js (hunk 2 above)
Comment (AnthonyMadia) [reply packages/frontend/src/__tests__/signupPage.js:54]: Not sure I am understanding where exactly the variable is needed. The `currently UniRep Social...` and `join us` regex are what the page renders and I don't think there's a way to refactor the `textbox` here.

Comment (vimwitch) [reply packages/frontend/src/__tests__/signupPage.js:54]: Ah sorry i meant the line below. `lorem ipsum dolor sit amet`.

Comment (AnthonyMadia) [reply packages/frontend/src/__tests__/signupPage.js:54]: Gotcha, will implement changes.

**Pull request merged**